waspc/data/Generator/templates/server/src/jobs/PgBossJobFactory.js

```diff
@@ -0,0 +1,43 @@
+import PgBoss from 'pg-boss'
+
+export const boss = new PgBoss({ connectionString: process.env.DATABASE_URL })
+
+export async function startPgBoss() {
+  console.log('Starting PgBoss...')
+  boss.on('error', error => console.error(error))
+  await boss.start()
+  console.log('PgBoss started!')
+}
+
+class PgBossJobFactory {
+  constructor(values) {
+    this.perform = () => { }
+    this.options = {}
+    this.startAfter = 0
+    this.jobName = 'unknown'
+    Object.assign(this, values)
+  }
+
+  delay(startAfter) {
+    return new PgBossJobFactory({ ...this, startAfter })
+  }
+
+  async submit(payload, options) {
```

Comment:
Renamed `performAsync()` to `submit()`. Sound ok?

Reply:
Soudns good to me!